### PR TITLE
Global css and routes

### PR DESCRIPTION
**Pull request by matzztam**

### CODE REVIEW

File: src/pages/RepositoryPage.vue [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<template>
+  <q-page class="text-center text-h4">
+    This is Login!
```
Comment (fencim) [inline src/pages/RepositoryPage.vue:3]: repository

File: src/router/routes.js [hunk 2]
```diff
@@ -1,29 +1,34 @@
...
     component: MainLayout,
     children: [
       {
         path: "login",
         name: "login-page",
         component: Login,
+      },
+      {
+        path: "RepositoryPage",
+        name: "Repository-page",
```
Comment (fencim) [inline src/router/routes.js]: use small  case kebab

**Review by fencim — CHANGES_REQUESTED** [review]

File: src/router/routes.js (hunk 2 above)
Comment (matzztam) [reply src/router/routes.js]: done sir i updated the route name

File: src/pages/RepositoryPage.vue (hunk 1 above)
Comment (matzztam) [reply src/pages/RepositoryPage.vue:3]: done. new PR.

Comment (fencim) [reply src/pages/RepositoryPage.vue:3]: you should have comitted and pushed in this branch rather than creating new PR in separate branch...

**Review by fencim — APPROVED** [review]

**Pull request merged**